help/sources/tutorials/ui/create/crm/sap-hybris-subscription-billing-customers-and-contacts.md

```diff
@@ -0,0 +1,249 @@
+---
+title: Create a SAP Hybris source connection in the UI
+description: Learn how to create a SAP Hybris source connection using the Adobe Experience Platform UI.
+exl-id: 45840d7e-4c19-4720-8629-be446347862d
+---
+# (Beta) Create a [!DNL SAP Hybris] source connection in the UI
+
+>[!NOTE]
+>
+>The [!DNL SAP Hybris] source is in beta. See the [sources overview](../../../../home.md#terms-and-conditions) for more information on using beta-labeled sources.
+
+The following tutorial walks you through the steps to create a [!DNL SAP Hybris] source connection to bring [[!DNL SAP] Subscription Billing](https://www.sap.com/products/financial-management/subscription-billing.html) contacts and customer data using the Adobe Experience Platform user interface.
+
+## Getting started {#getting-started}
+
+This tutorial requires a working understanding of the following components of Experience Platform:
+
+* [[!DNL Experience Data Model (XDM)] System](../../../../../xdm/home.md): The standardized framework by which [!DNL Experience Platform] organizes customer experience data.
+  * [Basics of schema composition](../../../../../xdm/schema/composition.md): Learn about the basic building blocks of XDM schemas, including key principles and best practices in schema composition.
+  * [Schema Editor tutorial](../../../../../xdm/tutorials/create-schema-ui.md): Learn how to create custom schemas using the Schema Editor UI.
+* [[!DNL Real-Time Customer Profile]](../../../../../profile/home.md): Provides a unified, real-time consumer profile based on aggregated data from multiple sources.
+
+If you already have a valid [!DNL SAP Hybris] account, you may skip the remainder of this document and proceed to the tutorial on [configuring a dataflow](../../dataflow/crm.md).
+
+### Gather required credentials {#gather-credentials}
+
+In order to connect [!DNL SAP Hybris] to Platform, you must provide values for the following connection properties:
+
+| Credential | Description |
+| --- | --- |
+| `Client Id` | The value of `clientid` from the service key. |
+| `Client Secret` | The value of `clientsecret` from the service key. |
+| `tokenEndpoint` | The value of `url` from the service key, it will be similar to `https://subscriptionbilling.authentication.eu10.hana.ondemand.com`. |
+| `region` | Your data center location. The region is present in the `url` and has a value similar to `eu10` or `us10`. For example if the `url` is `https://eu10.revenue.cloud.sap/api` you will need `eu10`. |
+	
+Refer to the [!DNL SAP Hybris] [documentation](https://help.sap.com/docs/CLOUD_TO_CASH_OD/987aec876092428f88162e438acf80d6/c5fcaf96daff4c7a8520188e4d8a1843.html) if you need any guidance.
```

Comment:
reformat link and text to: "For more information, please refer to the [[!DNL SAP Hybris] documentation](https://help.sap.com/docs/CLOUD_TO_CASH_OD/987aec876092428f88162e438acf80d6/c5fcaf96daff4c7a8520188e4d8a1843.html)."